docs/usage.md

```diff
@@ -135,14 +127,26 @@ You can also generate such `YAML`/`JSON` files via [nf-core/launch](https://nf-c
 
 ### Imputation modes
 
-You can choose different software to perform the imputation.
+You can choose different software to perform the imputation. In the following sections, the typical commands for running the pipeline with each software are included.
 
 #### QUILT
 
-The typical command for running the pipeline with this software is as follows:
+```bash
+nextflow run nf-core/phaseimpute --input samplesheet.csv --panel samplesheet_reference.csv --step impute --tool quilt --outdir results --genome GRCh37 -profile docker
+```
+
+#### STITCH
+
+```bash
+nextflow run nf-core/phaseimpute --input samplesheet.csv --step impute --tool stitch --outdir results --genome GRCh37 -profile docker
+```
+
+Notice that no reference panel is needed when running STITCH.
```

Comment:
You said that a reference panel is not needed but the preparation of the parameters require:
"VCF_SITES_EXTRACT_BCFTOOLS.out.panel_sites". How can we manage that ?

Reply:
There should be a `--posfile` parameter where users can input their own sites. This means that when running `--mode impute`, it should not run the `panelprep` mode (it is currently doing that for all tools by default).

If a tool needs the `panelprep` mode, it would be better to specify that by using `--mode panelprep,impute`.